pom.xml

```diff
@@ -13,6 +13,11 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <java.level>6</java.level>
```

Comment:
The problem is that 1.626 is greater in theory to 1.625.3 but not in practice, so a user could install the plugin in an instance without the needed features to run it if you use 1.625.3.